app/src/androidTest/java/com/github/geohunt/app/model/database/TestFirebaseChallengeRef.kt

```diff
@@ -50,5 +40,5 @@ class TestFirebaseChallengeRef {
             assertThat(challenge.correctLocation.latitude, closeTo(43.880433, 1e-5))
             assertThat(challenge.correctLocation.longitude, closeTo(-103.453748, 1e-5))
         }
-    }
+    }*/
```

Comment:
Pro-tip: Removing the entire test make it less visible and the one reviewing the PR will likely less notice 🙃

Reply:
The whole file is going to be deleted anyway, normally.